configs/FLYCOLORH743/config.h

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        FLYCOLORH743
+#define MANUFACTURER_ID   FLCO
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
```

Comment:
The schematic only shows ICM42688p. Will this come in a MPU6000 version? If so, we should create a separate target so the external clock can be defined for the 42688.